rubicon/objc/runtime.py

```diff
@@ -1739,8 +1734,10 @@ def py_from_ns(nsobj, *, _auto=False):
     elif _auto:
         # _auto is a private kwarg that is only passed when py_from_ns is called to perform an implicit conversion
         # of a method return value or argument into Python. In this case we only want to perform a few simple
-        # conversions (strings and numbers).
+        # conversions (boxed numbers to their Python equivalents).
         return nsobj
+    elif nsobj.isKindOfClass(NSString):
+        return str(nsobj)
```

Comment:
It took me a little bit to work out the significance of this change. It would be helpful to drop a comment noting that everything below the `elif _auto` are the data types that will be converted if you explicitly request conversion to Python, but won't be as part of an automated conversion. The existence of that "_auto" path wasn't a new addition, but it wasn't immediately obvious to me reviewing this, so since you're in the area, might as well clarify for next time.